src/main/java/mate/academy/Main.java

```diff
@@ -51,5 +58,21 @@ public static void main(String[] args) {
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
                         fastAndFurious.getId(), LocalDate.now()));
+
+        AuthenticationService authenticationService = (AuthenticationService) injector
+                .getInstance(AuthenticationServiceImpl.class);
+
+        try {
+            authenticationService.register("bob.marley@gmail.com", "password");
+        } catch (RegistrationException e) {
+            throw new RuntimeException("Could not register user", e);
+        }
+
+        try {
+            authenticationService.login("bob.marley@gmail.com", "password");
+        } catch (AuthenticationException e) {
+            throw new RuntimeException("Could not login user", e);
+        }
+
```

Comment:
What about when we trying to register when email address is taken? Please add this case.